boards/renesas/ek_ra4m2/doc/index.rst

```diff
@@ -61,7 +61,7 @@ available internal to the RA MCU
 **Special Feature Access**
 - 32 MB (256 Mb) External Quad-SPI Flash
 
-.. figure:: ek-ra4m2-board.webp
+.. figure:: ek_ra4m2.webp
```

Comment:
@kartben : Sorry, I haven't understood the problem here.
"boards/< vendor >/<board_name>/"
Is it required to change the name of image is same with <board_name>?
Is my understanding correct?

Reply:
Yes. It's a convention that's important as it's what allows to know, in case there are multiple images for a board, which one actually depicts the board. See https://github.com/zephyrproject-rtos/zephyr/pull/79160 and how this particular board shows up with a "wrong" image. Thanks!